tests/sharing_test.py

```diff
@@ -0,0 +1,166 @@
+import unittest
+
+from tests.uid2_token_generator import UID2TokenGenerator, Params
+from uid2_client import *
+import base64
+
+_master_secret = bytes(
+    [139, 37, 241, 173, 18, 92, 36, 232, 165, 168, 23, 18, 38, 195, 123, 92, 160, 136, 185, 40, 91, 173, 165, 221, 168,
+     16, 169, 164, 38, 139, 8, 155])
+_site_secret = bytes(
+    [32, 251, 7, 194, 132, 154, 250, 86, 202, 116, 104, 29, 131, 192, 139, 215, 48, 164, 11, 65, 226, 110, 167, 14, 108,
+     51, 254, 125, 65, 24, 23, 133])
+_master_key_id = 164
+_site_key_id = 165
+_test_site_key_id = 166
+_site_id = 9000
+_site_id2 = 2
+
+_example_id = 'ywsvDNINiZOVSsfkHpLpSJzXzhr6Jx9Z/4Q0+lsEUvM='
+_now = dt.datetime.now(tz=timezone.utc)
+_master_key = EncryptionKey(_master_key_id, -1, _now - dt.timedelta(days=-1), _now, _now + dt.timedelta(days=1),
+                            _master_secret, keyset_id=9999)
+_site_key = EncryptionKey(_site_key_id, _site_id, _now - dt.timedelta(days=-1), _now, _now + dt.timedelta(days=1),
+                          _site_secret, keyset_id=99999)
+
+_client_secret = "ywsvDNINiZOVSsfkHpLpSJzXzhr6Jx9Z/4Q0+lsEUvM="
+_example_uid = "ywsvDNINiZOVSsfkHpLpSJzXzhr6Jx9Z/4Q0+lsEUvM="
```

Comment:
_example_id,_client_secret,_example_uid   probably should be different strings :P